test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -38,37 +40,98 @@ class CheckYourAnswersControllerSpec extends SpecBase with SummaryListFluency wi
 
     "for a GET" - {
 
-      "must return OK and the correct view" in {
+      "must return OK and the correct view with valid non-optional data" in {
```

Comment:
nitpick: maybe more intuitive to state with mandatory data rather than non-optional :D

Reply:
agreed